aten/src/ATen/test/cuda_distributions_test.cu

```diff
@@ -119,7 +119,6 @@ TEST(DistributionsTest, TestPhiloxIncrementBigUniformTensor) {
   // calculate maximum number of threads that can be launched
   // and set the numel to be 8 times that
   const int block_size = 256;
-  dim3 dim_block(block_size);
```

Comment:
This is just a general cleanup or important?

Reply:
A general cleanup, should be moved to another PR.